keylime/tpm1.py

```diff
@@ -813,6 +813,18 @@ def check_quote(self,agent_id,nonce,data,quote,aikFromRegistrar,tpm_policy={},im
 
         return self.check_pcrs(agent_id,tpm_policy,pcrs,data,False,ima_measurement_list,ima_whitelist)
 
+    def sim_extend(self,hashval_1,hashval_0=None):
```

Comment:
Is this PR based on an outdated branch?  This patch should already be in `master` ...